tests/charts/test_pgbouncer.py

```diff
@@ -518,6 +518,18 @@ def test_extra_ini_configs(self):
         assert "server_round_robin = 1" in ini
         assert "stats_period = 30" in ini
 
+    def test_pgbouncer_deployment_extra_annotations(self):
```

Comment:
Can you add coverage [here](https://github.com/apache/airflow/blob/0fe73c2e8c0ca90faf54b39139ffab5769f4cca1/tests/charts/test_annotations.py#L371) instead?